Sources/Receipt/ReceiptTransactionTableViewCell.swift

```diff
@@ -33,30 +33,119 @@ final class ReceiptTransactionTableViewCell: UITableViewCell {
     private var iconColor: UIColor!
     private var iconBackgroundColor: UIColor!
     private let credit = HyperwalletReceipt.HyperwalletEntryType.credit.rawValue
-    // MARK: Life cycle
+
+    lazy var receiptTypeLabel: UILabel = {
+        let label = UILabel(frame: .zero)
+        label.font = Theme.Label.bodyFontMedium
+        label.textColor = Theme.Label.color
+        label.accessibilityIdentifier = "ReceiptTransactionTypeLabel"
+        return label
+    }()
+
+    lazy var amountLabel: UILabel = {
+        let label = UILabel(frame: .zero)
+        label.font = Theme.Label.bodyFontMedium
+        label.textAlignment = .right
+        label.accessibilityIdentifier = "ReceiptTransactionAmountLabel"
+        return label
+    }()
+
+    lazy var createdOnLabel: UILabel = {
+        let label = UILabel(frame: .zero)
+        label.font = Theme.Label.captionOne
+        label.textColor = Theme.Label.subTitleColor
+        label.accessibilityIdentifier = "ReceiptTransactionCreatedOnLabel"
+        return label
+    }()
+
+    lazy var currencyLabel: UILabel = {
+        let label = UILabel(frame: .zero)
+        label.font = Theme.Label.captionOne
+        label.textColor = Theme.Label.subTitleColor
+        label.textAlignment = .right
+        label.accessibilityIdentifier = "ReceiptTransactionCurrencyLabel"
+        return label
+    }()
+
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
-        super.init(style: .value1, reuseIdentifier: reuseIdentifier)
+        super.init(style: .default, reuseIdentifier: reuseIdentifier)
+        defaultInit()
     }
 
     required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
+        defaultInit()
+    }
+
+    private func defaultInit() {
+        let line1StackView = UIStackView(frame: .zero)
+        setup(line1StackView, with: receiptTypeLabel, and: amountLabel)
+
+        let line2StackView = UIStackView(frame: .zero)
```

Comment:
rename line2StackView to subTitleStackView

Reply:
fixed